src/pages/en/tutorial/3-components/1.md

```diff
@@ -87,18 +89,70 @@ This allows you to get started quickly with any working code, often duplicated t
 
 ### Test your knowledge
 
-Use these words in the text below. Click the blanks to reveal if you're right!
 
-<Blanks>
-- reusable
-- `src/pages`
-- refactor
-- standard HTML elements
-</Blanks>
-
-`.astro` files can use || **standard HTML elements** || and other Astro components in the same template, alongside each other. This means that you can decide when and where to || **refactor** || your code, creating || **reusable** || fragments of HTML defined in an Astro component. 
-
-These components should live in a folder outside of || **`src/pages`** || since they are not meant to be pages on your website.
+1. Astro files can use these alongside Astro components in their template:
+  <MultipleChoice>
+    <Option>
+      reusable
+    </Option>
+    <Option>
+      `src/pages/`
+    </Option>
+    <Option>
+       refactor
+    </Option>
+    <Option isCorrect>
+       standard HTML elements
+    </Option>
+  </MultipleChoice>
+
+2. You can do this when you have elements repeated on multiple pages:
+  <MultipleChoice>
+      <Option>
+        reusable
+      </Option>
+      <Option>
+        `src/pages/`
+      </Option>
+      <Option isCorrect>
+        refactor
+      </Option>
+      <Option>
+        standard HTML elements
+      </Option>
+    </MultipleChoice>
```

Comment:
4 questions with 4 options each seems a lot to me and 2 & 3 here are very similar. How do you feel about condensing to 3 questions with 3 options each?

```suggestion
2. You can do this when you have elements repeated on multiple pages:
    <MultipleChoice>
      <Option>
        copy/paste each time you need the elements
      </Option>
      <Option isCorrect>
        refactor to use reusable components
      </Option>
      <Option>
        use `<style>` to change their appearance
      </Option>
    </MultipleChoice>
```